ci/client/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 # Used to ensure that SPL programs are buildable by external clients
 
 [dependencies]
-solana-sdk = "1.2"
+solana-sdk = "1.2.17"
```

Comment:
@jackcmay - fyi I changed 1.2 -> 1.2.17 because we don't want anything less than 1.2.17